Automattic/wp-calypso
# Fix/jetpack manage error output

Currently the output of the Jetpack manage error page is not very accurate. 
Before:
![screen shot 2015-12-16 at 20 16 02](https://cloud.githubusercontent.com/assets/115071/11861128/ef5c0d30-a431-11e5-8ce5-c147bf52e926.png)
![screen shot 2015-12-16 at 20 16 11](https://cloud.githubusercontent.com/assets/115071/11861130/f133bac2-a431-11e5-95d0-a519506201a7.png)
![screen shot 2015-12-16 at 20 16 22](https://cloud.githubusercontent.com/assets/115071/11861131/f40fb96c-a431-11e5-8c6c-35d344856601.png)

After:
![screen shot 2015-12-16 at 20 17 22](https://cloud.githubusercontent.com/assets/115071/11861143/16562ea2-a432-11e5-954d-3325c2ca6dd8.png)
![screen shot 2015-12-16 at 20 17 31](https://cloud.githubusercontent.com/assets/115071/11861145/18cf634c-a432-11e5-9d41-0dec3bce7742.png)

![screen shot 2015-12-16 at 20 17 15](https://cloud.githubusercontent.com/assets/115071/11861141/1310e692-a432-11e5-94a4-0b8ed63e54d9.png)


**To test:**
On a site that you have jetpack manage disabled visit:
Theme, menus, security setting, plugins, and the plugin browser and individual plugin
And check that the message makes sense. 
Does the 'enable manage' button take you to the desired section in on the jetpack site. 

Does it cause any error? 

cc: @johnHackworth, @lezama, @ebinnion, @alternatekev
